packages/api/src/controllers/organicCertifierSurveyController.js

```diff
@@ -161,8 +161,17 @@ const organicCertifierSurveyController = {
         });
       }
       const organicCertifierSurvey = await knex('organicCertifierSurvey')
-        .where({ farm_id })
+        .where({ farm_id, interested: true })
         .first();
+
+      // Skip the whole flow in case this Farm is not pursuing any cert.
+      if (organicCertifierSurvey === undefined) {
+        // Status code 202 Accepted https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+        return res
+          .status(202)
```

Comment:
The doc says "`202 Accepted` response status code indicates that the request has been accepted for processing, **but the processing has not been completed**", but the processing has been completed here. `204` seems suitable to me.

@Duncan-Brain @kathyavini what do you think?

Reply:
For the error code:

I almost expect an error code in the 400's tbh haha. 

I agree with Sayakas point about processing not being completed. And 202 is non-committal to the outcome of the processing described in the Mozilla description. But we do know exactly why it fails to return an export document.

For the premise of the ticket:

I wonder if you actually should still maybe be able to export documents even if you are not interested. My understanding is we are capable of returning something for any farm -- for example creating an "other" certification type.

Reply:
@SayakaOno @Duncan-Brain thanks for the feedback, would you mind sharing what status code should be used? 
204 or in the 400s range.
To me is pretty difficult to say this is an error because it looks like the user gave the consent of not being interested on certifications so the backend should inform that through the frontend, not as an error but some sort of goodmoodhappy reminder (204?). 
Also thinking it depends on the team taste on how to deal with this situation so I'll update the PR according your expectations. 

About this:
> I wonder if you actually should still maybe be able to export documents even if you are not interested. My understanding is we are capable of returning something for any farm -- for example creating an "other" certification type.

Is this something that needs some designing within the core team ? Not sure about the discussions around this topic.

Reply:
Ah sorry, I was slow to notice this tag. I would say it's a `400` response code. There is no intentional frontend flow that hits this API endpoint with `interested` set to false. If it were to happen via the frontend, it would be an error from the client for making that request.

> My understanding is we are capable of returning something for any farm -- for example creating an "other" certification type.

@Duncan-Brain yeah there is a working flow for no supported certifier / "generic forms" -- is that the situation you mean? That flow also sets `interested` to true though, so it should still be compatible with adding this API check.

Reply:
@thxv3n0lvl 
> To me is pretty difficult to say this is an error because it looks like the user gave the consent of not being interested on certifications so the backend should inform that through the frontend, not as an error but some sort of goodmoodhappy reminder (204?).

I agree with you. But let's go with `400` this time. Thank you again!